src/test/fuzz/banman.cpp

```diff
@@ -38,47 +38,34 @@ FUZZ_TARGET_INIT(banman, initialize_banman)
     {
         BanMan ban_man{banlist_file, nullptr, ConsumeBanTimeOffset(fuzzed_data_provider)};
         while (fuzzed_data_provider.ConsumeBool()) {
-            switch (fuzzed_data_provider.ConsumeIntegralInRange<int>(0, 11)) {
+            switch (fuzzed_data_provider.ConsumeIntegralInRange<int>(0, 6)) {
             case 0: {
-                ban_man.Ban(ConsumeNetAddr(fuzzed_data_provider),
-                    ConsumeBanTimeOffset(fuzzed_data_provider), fuzzed_data_provider.ConsumeBool());
-                break;
-            }
-            case 1: {
```

Comment:
After you rebase, you may want to apply this patch: https://github.com/bitcoin/bitcoin/issues/20837#issuecomment-760097060

Reply:
Done in 0dbe88ad3894cd76d52ed196fd2eed39de1296d2